### DISCUSSION TITLE
How to use global variable

### DISCUSSION THREAD

**Discussion by akbarism** [vuejs/pinia#1366] [Help and Questions]
so i use dayjs on my vue project, and declare it as a global variable
this is my configuration :

on dayjs.js :
`
import dayjs from "dayjs";`
`export const dayConfig = (app) => {
  app.provide("day", dayjs);
};
`

on main,js : 

`import { dayConfig } from "./plugins/Dayjs";`
`dayConfig(app);`

in component.vue I use it like this :

`import { inject } from "vue";`
`const day = inject("day");`
`const date = day().format('YYYY-MM-DD')`
if I use that method in store.js it will display a warning: 
inject() can only be used inside setup() or functional components.

**Comment by posva** [comment]
Just import dayjs (everywhere). You don’t need to provide that kind of variables
